tests/suites/test_suite_x509parse.function

```diff
@@ -419,7 +456,7 @@ exit:
 
 /* BEGIN_CASE depends_on:MBEDTLS_FS_IO:MBEDTLS_X509_CRT_PARSE_C:MBEDTLS_X509_CRL_PARSE_C:MBEDTLS_X509_TRUSTED_CERTIFICATE_CALLBACK */
 void x509_verify_ca_cb_failure( char *crt_file, char *ca_file, char *name,
-                                int exp_ret, char *exp_vrfy_out )
+                                int exp_ret )
```

Comment:
Can we also have `exp_flags` and test them?

Reply:
Not changed, should this be added as a task to future?